README.md

```diff
@@ -1,58 +1,66 @@
+<div align="center">
+  <a href="https://github.com/webpack/webpack">
+    <img width="200" height="200" src="https://webpack.js.org/assets/icon-square-big.svg">
+  </a>
+</div>
+
 [![npm][npm]][npm-url]
+[![node][node]][node-url]
 [![deps][deps]][deps-url]
-[![test][test]][test-url]
-[![coverage][cover]][cover-url]
+[![tests][tests]][tests-url]
 [![chat][chat]][chat-url]
+[![size][size]][size-url]
 
-<div align="center">
-  <a href="https://webpack.js.org/">
-    <img width="200" height="200" vspace="" hspace="25" src="https://cdn.rawgit.com/webpack/media/e7485eb2/logo/icon-square-big.svg">
-  </a>
-  <h1>webpack-defaults</h1>
-  <p>Defaults to be shared across webpack projects</p>
-</div>
+# webpack-defaults
+
+Project configuration and boilerplate defaults for webpack projects
+
+## Requirements
+
+This module requires a minimum of Node v6.9.0 and Webpack v4.0.0.
+
+## Getting Started
 
-<h2 align="center">Install</h2>
+To begin, you'll need to install `webpack-defaults`:
 
-```bash
-npm install --save-dev webpack-defaults
+```console
+$ npm install @webpack-contrib/defaults --save-dev
 ```
 
-This will write a npm script you can run through
+After install a `defaults` NPM script will be written to the local
+`package.json`. To sync a project with webpack-defaults, simply run:
 
-```bash
+```
 npm run defaults
 ```
 
-Any time you want to sync your project with the defaults, update this dependency and run the command again.
+## Contributing
+
+Please take a moment to read our contributing guidelines if you haven't yet done so.
+
+#### [CONTRIBUTING](./.github/CONTRIBUTING)
 
-<h2 align="center">Maintainers</h2>
+## License
 
-<table>
-  <tbody>
-    <tr>
-      <td align="center">
-        <a href="https://github.com/sapegin">
-          <img width="150" height="150" src="https://avatars.githubusercontent.com/u/70067?v=3">
-          </br>
-          Artem Sapegin
-        </a>
-      </td>
-    </tr>
-  <tbody>
-</table>
```

Comment:
I think we don't should remove this. Looks webpack docs use this section, need check or maybe ping something.